test/check-ostree

```diff
@@ -632,7 +632,7 @@ class OstreeCase(MachineCase):
         b.wait_in_text(".pf-c-empty-state__body", "Not authorized")
         self.assertIn("Reconnect", b.text(".pf-c-empty-state button"))
 
-        b.become_superuser()
+        b.become_superuser(passwordless=m.image == "rhel4edge")
```

Comment:
Can you please explain this change?

Reply:
edge sets up passwordless sudo. Should I add comment into the code?